### PR TITLE
Protactor End to End Testing

### PR DESCRIPTION
**Pull request by leocaseiro**
Add http://www.protractortest.org/ End to End testing for AngularJS, Angular4 and so on...

### CODE REVIEW

File: static/logos.json [hunk 1]
```diff
@@ -537,6 +537,9 @@
     "name": "PrimeNG"
   },
   {
+    "name": "Protractor"
```
Comment (samiheikki) [inline static/logos.json]: Can you move this to after `Prototype` to maintain alphabetic order.

**Review by samiheikki — CHANGES_REQUESTED** [review]

File: static/logos.json (hunk 1 above)
Comment (leocaseiro) [reply static/logos.json]: I apologise. I always think it spells `Protactor`.

File: static/logos.json [hunk 2]
```diff
@@ -540,6 +540,9 @@
     "name": "Prototype"
   },
   {
+    "name": "Protractor"
+  },
+  {
```
Comment (leocaseiro) [inline static/logos.json:545]: Thanks for pointing that out.

I fixed now @samiheikki

Comment (samiheikki) [reply static/logos.json:545]: Thanks. There now seems to be extra file included `angular-protractor.png`. Can you remove that?

Comment (leocaseiro) [reply static/logos.json:545]: Hi, @samiheikki. The file is removed.

**Review by samiheikki — APPROVED** [review]

File: static/logos.json (hunk 2 above)
Comment (samiheikki) [reply static/logos.json:545]: Thank you!

Comment (leocaseiro) [reply static/logos.json:545]: Thanks for building this awesome game and for approving this PR 😄

**Pull request merged**